### ISSUE TITLE
problem with fires lasting forever

### ISSUE THREAD

**Issue by Tripform** [codengine/SOTFEdit#19]
Hello,

Thanks for the awesome save game editor.  Quick note.  When using the refuel and ignite fires command.  The fires stay lit forever. Is there a way to manage the amount of fuel that is added to the fires?  The only way to remove fires is to turn on building destruction.  And sometimes players might not want to have to smash there building apart to remove a perpetual fire.

Thank you!

**Comment by codengine** [comment]
Thanks for the input,
I'll add a reset function along the changes introduced with Patch 14 in the next update.

**Comment by codengine** [comment]
The feature has been added in v0.11.16: https://github.com/codengine/SOTFEdit/releases/tag/v0.11.16

**Comment by Tripform** [comment]
Not sure its working, should i send my save file for reference?

**Comment by codengine** [comment]
The fires are reset to 1 fuel per (minute? second?) which is default and 300 fuel and they are ignited.

Maybe you just have to wait a bit :)

**Comment by Tripform** [comment]
I will check but last night had a two or three hour play session and the fires never did go out. Thanks,Pablo Manzarek On Nov 13, 2023, at 11:54 PM, Stefan Hueg ***@***.***> wrote:﻿
Reopened #19.

**Comment by codengine** [comment]
I'll try to reproduce it later today. Which type of fire did you test it with?

**Comment by Tripform** [comment]
It is doing this on all the different types. 1 stick, two sticks, rocks, and log fires. Doesn’t seem to make a difference. Also was thinking I could somehow go to the structures tab and delete out all the fires. But there is no category for that.  On Nov 14, 2023, at 10:06 AM, Stefan Hueg ***@***.***> wrote:﻿
I'll try to reproduce it later today. Which type of fire did you test it with?

**Comment by codengine** [comment]
So, I tried it with a fresh savegame, placed a minimal fire with two sticks and it is extinguished after five minutes.
When I set it to eternal, it will stay lit (of course).

When I reset it again, it will extinguish after five minutes. Could it be that the issue is that Steam overwrote your changes through a Cloud Save?

The best way to cancel this out is to start the game first, then edit the savegame (and save), then enter your savegame.

**Comment by Tripform** [comment]
I’ll try it out. I’m not sure about the steam cloud save overwriting. But everything else I do on sotfedit works as it should. I’ll get back to you if it doesn’t work Thanks,Pablo Manzarek On Nov 14, 2023, at 1:34 PM, Stefan Hueg ***@***.***> wrote:﻿
So, I tried it with a fresh savegame, placed a minimal fire with two sticks and it is extinguished after five minutes.
When I set it to eternal, it will stay lit (of course).
When I reset it again, it will extinguish after five minutes. Could it be that the issue is that Steam overwrote your changes through a Cloud Save?
The best way to cancel this out is to start the game first, then edit the savegame (and save), then enter your savegame.

**Comment by Tripform** [comment]
Hello, I’m can confirm that the issue seems to be fixed!  Thanks for the quick responses!  Cheers to you. On Nov 14, 2023, at 1:40 PM, Pablo Manzarek ***@***.***> wrote:﻿I’ll try it out. I’m not sure about the steam cloud save overwriting. But everything else I do on sotfedit works as it should. I’ll get back to you if it doesn’t work Thanks,Pablo Manzarek On Nov 14, 2023, at 1:34 PM, Stefan Hueg ***@***.***> wrote:﻿
So, I tried it with a fresh savegame, placed a minimal fire with two sticks and it is extinguished after five minutes.
When I set it to eternal, it will stay lit (of course).
When I reset it again, it will extinguish after five minutes. Could it be that the issue is that Steam overwrote your changes through a Cloud Save?
The best way to cancel this out is to start the game first, then edit the savegame (and save), then enter your savegame.
